hack/deployer/README.md

```diff
@@ -1,71 +1,51 @@
 # Deployer
 
-Deployer is the provisioning tool that aims to be the interface to multiple Kubernetes providers.
+Deployer is the provisioning tool that aims to be the interface to multiple Kubernetes providers. Currently, it supports GKE and AKS.
 
-At the moment it support GKE and AKS.
+# Typical usage
 
-Default values for settings are kept in config/plans.yml file. More settings and setting overrides are provided via deployer-config.yml file.
+## Provision
 
-## Typical usage
+* GKE
 
-While in repo root, run once:
-```
-make dependencies
-cd hack/deployer
-go build
-```
+  * Install [Google Cloud SDK](https://cloud.google.com/sdk/install)
+  * Install Google Cloud SDK beta components by running `gcloud components install beta`
+  * Make sure that container registry authentication is correctly configured as described [here](https://cloud.google.com/container-registry/docs/advanced-authentication)
+  * Set `GCLOUD_PROJECT` to the name of the GCloud project you wish to use
+  * Run from the [project root](/):
+    ```bash
+    make switch-gke bootstrap-cloud
+    ```
 
-Then, depending on the provider, run the following:
+* AKS
 
-For GKE, with your GCLOUD_PROJECT:
-```
-cat > config/deployer-config.yml << EOF
-id: gke-dev
-overrides:
-  clusterName: dkowalski-dev-cluster
-  gke:
-    gCloudProject: GCLOUD_PROJECT
-EOF
-```
+  * Install [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+  * Set `RESOURCE_GROUP` to the name of the Resource Group you wish to deploy in
+  * Set `ACR_NAME` to the name of Azure Container Registry you wish to use
+  * Run from the [project root](/):
+    ```bash
```

Comment:
nit: code blocks are usually surrounded by new lines

Reply:
Surrounding this by new lines causes the list to go from nice:
<img width="595" alt="Screenshot 2020-02-18 at 15 39 33" src="https://user-images.githubusercontent.com/50632861/74784149-98839e00-52a7-11ea-8aa1-12d608c6ad40.png">
to (IMO) less nice:
<img width="586" alt="Screenshot 2020-02-18 at 15 39 45" src="https://user-images.githubusercontent.com/50632861/74784157-9ae5f800-52a7-11ea-8b64-25abbc010971.png">